content/index.fr.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Vue d'ensemble des standards de l'accessibilité"
+permalink: /fundamentals/fr
+ref: /fundamentals/
+lang: fr
+last_updated: 2020-12-28
+translators:
+- name : "Sofia Ahmed"
+contributors:
+- name : "Sandra Velarde Gonzalez (ETNIC)"
+github:
+  repository: w3c/wai-fundamentals-overview
+  path: content/index.fr.md
+footer: > # Text in footer in HTML
+  <p><strong>Date :</strong> Mise à jour : 19 août 2020.</p>
+  <p><strong>Rédactrice :</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Développé par le Groupe de travail Éducation et Promotion (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+
+---
+
+Les ressources suivantes contiennent des introductions à l'accessibilité :
+
+[[Introduction à l'accessibilité Web]](/fundamentals/accessibility-intro/)
+:   Introduit à l'accessibilité Web et renvoie à des ressources supplémentaires.
+    -   [[Vidéo : introduction à l'accessibilité Web et aux standards du W3C (4 minutes)]](/videos/standards-and-benefits/) .
+
+[[Éléments essentiels de l'accessibilité Web]](/fundamentals/components/)
+:   Montre comment l'accessibilité Web dépend de l'association entre les nombreuses composantes du développement Web et de l'interaction utilisateur, et le lien entre les différentes règles de la WAI : les Règles pour l'accessibilité des contenus Web ([WCAG](/standards-guidelines/wcag/), les Règles d'accessibilité pour les outils d'édition ([ATAG](/standards-guidelines/atag/) et les Règles pour l’accessibilité des agents utilisateurs ([UAAG](/standards-guidelines/uaag/).  
+
+[[Principes d'accessibilité]](/fundamentals/accessibility-principles/)
+:   Aide les développeurs, les designers, et tout autre intervenant à comprendre les principes pour créer des sites et des applications Web, des navigateurs, et d'autres outils accessibles. Fournit des liens vers les standards internationaux de l'Initiative pour l'accessibilité du Web (WAI) du W3C et vers une page dédiée aux [témoignages d'utilisateurs du Web](/people-use-web/user-stories/).
+
+[[L'accessibilité Web illustrée : explorer l'impact et les bénéfices pour tous]](/perspective-videos/)
+:   **Vidéos** démontrant le caractère essentiel de l'accessibilité Web pour les personnes en situation de handicap et son utilité pour tous. Les vidéos suivantes exemplifient l'impact de l'accessibilité et les bénéfices pour tous dans diverses situations. Chaque vidéo dure environ 1 minute et la [compilation](https://www.youtube.com/watch?v=3f31oufqFSM) dure 7 min 37 s. Chaque page contient des informations supplémentaires.
+
+[[Comment les personnes handicapées utilisent le Web]](/people-use-web/)
+:   Introduit comment les personnes porteuses de handicap utilisent le Web. Décrit les outils et les solutions pour lesquelles les personnes qui présentent différents types de handicaps optent pour naviguer sur le Web et les obstacles auxquels elles se heurtent lorsque le design n'est pas optimal.
```

Comment:
English: ... Describes tools and approaches that people with different kinds of disabilities use to browse the web and the barriers they encounter due to poor design.

French:... Décrit les outils et les solutions pour lesquelles les personnes qui présentent différents types de handicaps optent pour naviguer sur le Web et les obstacles auxquels elles se heurtent lorsque le design n’est pas optimal.

Comment:  I it better in French to use "approches" in the French version since we do not speak of "solutions" in the English. Second, is it possible to simplfy the second part of the sentence to "les obstacles recontrés" instead of "les obstacles auxquels elles se heurtent"

Reply:
Received from @remibetin a +1